### PR TITLE
[Feature] API 21 and 22 support

### PR DESCRIPTION
**Pull request by iaulakh**
## Purpose

* This PR is to support Android API 21 and 22.  We were not supporting API 21 and 22 because of theming issue, and this PR fixes the issue.
* Emulator crashes for API 21 and 22 (on call screen) - known issue

## Impact
* call join button, call end buttons
* Speaker selection icons
* video, mic mute/unmute icon
* setup screen disabled video icon color
* participant audio/video/pip background color
* Video view info header background color

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code
Please make sure theming is working fine from API 21+ onwards. 
If you do not have device, confirm with emulator on API 23+
Verify with dark and light theme

### CODE REVIEW

File: azure-communication-ui/azure-communication-ui/src/main/res/layout/azure_communication_ui_fragment_setup.xml [hunk 1]
```diff
@@ -170,7 +170,7 @@
 
         <com.microsoft.fluentui.widget.Button
             android:id="@+id/azure_communication_ui_setup_join_call_button"
-            style="@style/AzureCommunicationUICalling.Button"
```
Comment (phileo99) [inline azure-communication-ui/azure-communication-ui/src/main/res/layout/azure_communication_ui_fragment_setup.xml:173]: even though it's effectively an alias for `Widget.FluentUI.Button` at the moment,  it's still more flexible to retain this style definition so then if theming requirements ever change in future we only need to modify `azure_communication_ui_styles.xml` in one location

**Review by phileo99 — CHANGES_REQUESTED** [review]
tested on API25 and API 31
call join button, call end buttons
Speaker selection icons
video, mic mute/unmute icon
setup screen disabled video icon color
participant audio/video/pip background color
Video view info header background color

**Review by phileo99 — CHANGES_REQUESTED** [review]
tested on API25 and API 31
call join button, call end buttons
Speaker selection icons
video, mic mute/unmute icon
setup screen disabled video icon color
participant audio/video/pip background color
Video view info header background color

**Review by phileo99 — APPROVED** [review]
👍

**Review by ShaunaSong — APPROVED** [review]

**Pull request merged**